schemas/context/opportunity-contactitems.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/contactitems",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Opportunity Contact Items",
+  "type": "object",
+  "description": "Defines the list of contact items for a given opportunity",
+  "definitions": {
+    "contactitems": {
+      "properties": {
+        "xdm:roleId": {
+          "title": "Contact Role Id",
+          "description": "Contact role unique identifier.",
+          "type": "string"
+        },
+        "xdm:role": {
+          "title": "Contact Role",
+          "description": "Role of the opportunity contact like finance , sales .",
+          "type": "string"
+        },
+        "xdm:isPrimary": {
```

Comment:
why is this prefixed with is? We use xdm:primary for identities. Shouldn't we re-use this field given it has the same semantic meaning?

Reply:
removed is